__tests__/plugins/browser-console.test.ts

```diff
@@ -55,4 +55,49 @@ describe('BrowserConsole', () => {
     expect(testMessage.timestamp).toBeDefined();
     expect(testMessage.step).toEqual({ name: 'step-name', index: 0 });
   });
+
+  it('should capture browser page errors', async () => {
+    const driver = await Gatherer.setupDriver({ wsEndpoint });
+    const browserConsole = new BrowserConsole(driver);
+    const { page } = driver;
+    browserConsole.start();
+    await page.goto(server.TEST_PAGE);
+    browserConsole._currentStep = { name: 'step-name', index: 0 };
+    const bodyHandle = await page.$('body');
+    try {
+      await page.evaluate(
```

Comment:
You can do await page.setContent to make this simpler.

Reply:
done